lib/vendor/dynamicImport.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2018 uupaa and 2019 Google LLC
+ * Licensed under the MIT license.
+ *
+ * Copied from https://github.com/GoogleChromeLabs/dynamic-import-polyfill
+*/
+
+module.exports = function(module) {
+  function cleanup(script) {
+    URL.revokeObjectURL(script.src);
+    script.remove();
+  }
+
+  window.__twilioImportedModules = window.__twilioImportedModules || {};
+
+  return new Promise((resolve, reject) => {
+    const absURL = new URL(module, window.location);
+
+    if (window.__twilioImportedModules[absURL]) {
+      return resolve(window.__twilioImportedModules[absURL]);
+    }
```

Comment:
This code has some features from `https://github.com/GoogleChromeLabs/dynamic-import-polyfill` that we might not need. It stores each imported module in a window object `window.__twilioImportedModules`. If this function is called again for a module that was already imported, then it will just return that object instead of importing the file again.

This would help this function be used for different modules multiple times, but I don't think we need that functionality.